test/unit/mdc-select/mdc-select-enhanced.test.js

```diff
@@ -992,36 +995,41 @@ test('#constructor instantiates a leading icon if an icon element is present', (
   document.body.removeChild(menu);
 });
 
+
 test('#constructor instantiates the helper text if present', () => {
-  const container = getHelperTextFixture();
-  document.body.appendChild(container);
-  const component = new MDCSelect(container.querySelector('.mdc-select'));
+  const hasMockFoundation = true;
+  const hasMockMenu = true;
+  const hasLabel = true;
+  const hasOutline = false;
+  const hasHelperText = true;
+  const {container, helperText} = setupTest(hasLabel, hasOutline, hasMockFoundation, hasMockMenu,
+    hasHelperText);
 
-  assert.instanceOf(component.helperText_, MDCSelectHelperText);
+  assert.instanceOf(helperText, FakeHelperText);
```

Comment:
This line (as well as in mdc-select.test.js) will still need to check the component property - otherwise this is effectively just testing `setupTest`.